### PR TITLE
Adds Armstrong Style as a martial art and syndie kit.

### PR DESCRIPTION
**Pull request by karmaisblackandbluepilled**
## Changelog
:cl:
add: Brad Armstrong's brain was finally found in a burnt out ditch around a mass grave, and the Syndicate acquired solitary rights to distribute his famous martial art for all the operatives with enough TCs!
/:cl:

### CODE REVIEW

**Review by EdgeLordExe — COMMENTED** [review]
Ok lest do it less go

File: code/modules/mob/living/carbon/human/human_helpers.dm [hunk 1]
```diff
@@ -104,7 +104,7 @@
 			balloon_alert(src, "fingers are too big!")
 			return FALSE
 	if(HAS_TRAIT(src, TRAIT_NOGUNS))
-		to_chat(src, span_warning("You can't bring yourself to use a ranged weapon!"))
+		to_chat(src, span_warning("Use of ranged weaponry would bring dishonor to the clan."))
```
Comment (EdgeLordExe) [inline code/modules/mob/living/carbon/human/human_helpers.dm:107]: Not sure about this one.

File: code/modules/uplink/uplink_items.dm [hunk 2]
```diff
@@ -2126,3 +2126,10 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/melee/execution_sword
 	cost = 1 //it sucks balls for anything but memeing
 	surplus = 30 //Theres a good chance this will end up in surplus crates, so its a great way to add a little spice to any meme round.
+
+/datum/uplink_item/dangerous/armstrong
+	name = "Armstrong Style Kit"
+	desc = "A kit with the necessary equipment to become an excellent martial artist, and terrible parent!"
+	item = /obj/item/storage/box/syndie_kit/armstrong
+	cost = 12
```
Comment (EdgeLordExe) [inline code/modules/uplink/uplink_items.dm:2134]: May need to be adjusted to 14 tc

**Review by EdgeLordExe — APPROVED** [review]
Overall looks good

File: code/modules/uplink/uplink_items.dm (hunk 2 above)
Comment (karmaisblackandbluepilled) [reply code/modules/uplink/uplink_items.dm:2134]: yeah ok

File: code/modules/mob/living/carbon/human/human_helpers.dm (hunk 1 above)
Comment (karmaisblackandbluepilled) [reply code/modules/mob/living/carbon/human/human_helpers.dm:111]: ok changed

**Comment by karmaisblackandbluepilled** [comment]
FUCK I HATE MERGE CONFLICTS

**Pull request merged**